### DISCUSSION TITLE
How to open a file hyperlink(ed_grep) with a shortcut instead of Ctrl+Shift+LMB?

### DISCUSSION THREAD

**Discussion by Andrew15-5** [kovidgoyal/kitty#6888] [Q&A] [answered]
https://sw.kovidgoyal.net/kitty/kittens/hyperlinked_grep/:

> Hold down the <kbd>Ctrl</kbd>+<kbd>Shift</kbd> keys and click on any of the result lines, to open the file in vim at the matching line. If you use some editor other than vim, you should adjust the open-actions.conf file accordingly.

https://sw.kovidgoyal.net/kitty/open_actions/:

> ### Scripting the mouse click

I tried something like `map kitty_mod+;>o>u kitten hints --type hyperlink --program "kitty +open"`, but it doesn't support this link.

Am I missing something?

**Comment by kovidgoyal** [comment, marked as answer]
ctrl+shift+p>y

On Mon, Dec 04, 2023 at 09:38:12AM -0800, Andrew Voynov wrote:
> https://sw.kovidgoyal.net/kitty/kittens/hyperlinked_grep/:
> 
> > Hold down the <kbd>Ctrl</kbd>+<kbd>Shift</kbd> keys and click on any of the result lines, to open the file in vim at the matching line. If you use some editor other than vim, you should adjust the open-actions.conf file accordingly.
> 
> https://sw.kovidgoyal.net/kitty/open_actions/:
> 
> > ### Scripting the mouse click
> 
> I tried something like `map kitty_mod+;>o>u kitten hints --type hyperlink --program "kitty +open"`, but it doesn't support this link.
> 
> Am I missing something?
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/6888
> You are receiving this because you are subscribed to this thread.
> 
> Message ID: ***@***.***>

-- 
_____________________________________

Dr. Kovid Goyal 
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________

**Reply by Andrew15-5** [reply]
Oh, so it's just `kitten hints --type hyperlink` without `--program` ([link](https://sw.kovidgoyal.net/kitty/conf/#shortcut-kitty.Open-the-selected-hyperlink)). Thank you!
